op-guide/ansible-deployment.md

```diff
@@ -686,3 +686,16 @@ Ubuntu 系统可使用以下命令安装：
 $ sudo apt-get install python-pip
 $ sudo pip install jmespath
 ```
+
+#### 启动 Pump/Drainer 报 ```zk: node does not exist``` 错误
```

Comment:
- To format code within a sentence, use single backticks:
    `zk: node does not exist`
- To format code block, use triple backticks.

**Please also fix similar format mistakes involved in this PR.**

For details, see [Quoting code](https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code) guide.